extension/otto/gohan_core.go

```diff
@@ -217,7 +222,13 @@ func loadNPMModules() {
 				break
 			}
 			if !strings.HasSuffix(module, ".js") {
-				module = module + "/index.js"
+				if moduleFile, err := os.Stat(module); err == nil {
```

Comment:
not sure if this is correct computation, although it will work most of the time. IMO, we should follow npm spec and lookup package.json/main. See https://docs.npmjs.com/files/package.json#main , source https://github.com/nodejs/node/blob/master/lib/module.js